test/whitespace/strings.stmt

```diff
@@ -32,4 +32,36 @@ one
 two
 
 
-''';
\ No newline at end of file
+''';
+>>> trailing space multiline string
+var x = ''' \   \ \
+z''';
+<<<
+var x = '''
+z''';
+>>> trailing space multiline string, no newline
+var x = '''   \ ''';
+<<<
+var x = '''   \ ''';
+>>> trailing space multiline interpolation string
+var x = ''' \   \ \
+${''}''';
+<<<
+var x = '''
+${''}''';
+>>> trailing space multiline interpolation before newline string
+var x = '''   ${''} ''';
+<<<
+var x = '''   ${''} ''';
+>>> trailing space multiline raw string
+var x = r''' \   \ \
+z''';
+<<<
+var x = r'''
+z''';
+>>> trailing space multiline doublequotes
+var x = """ \   \ \
+z""";
+<<<
+var x = """
+z""";
```

Comment:
If you want to test the various kinds of whitespace characters, the tests support a custom little metasyntax for Unicode literals. Take a look at test/whitespace/unicode.txt for some examples.

Reply:
Will do.

Reply:
And done.